src/components/writeReviewButton/WriteReviewButton.js

```diff
@@ -53,6 +54,11 @@ const WriteReviewButton = (props) => {
     }
   };
 
+  const loginToReview = () => {
+    setFromReview(true);
+    handleShowLogInModal();
+  };
+
```

Comment:
This is a good solution, however not quite there all the way. If you press on login to review and then press register new user, you still have the option of continuing without logging in.